pkg/cmdlets/commit/commit.go

```diff
@@ -74,8 +74,7 @@ func newCommitOutput(commit *mono_models.Commit, orgs []gmodel.Organization, isL
 		StructuredChanges: structuredChanges,
 	}
 
-	commitOutput.Date = commit.AtTime.String()
```

Comment:
This is immediately changed on line 82 (81 in the new diff), so that's why I removed it here.